### ISSUE TITLE
Versatile weapon showing wrong damage bonus on output

### ISSUE THREAD

**Issue by sorrelfid** [Sklore/HL_DD_5e_Colab#544]
10th level fighter with a battle axe (versatile weapon).  In HL proper, the damage

 bonus is-- correctly-- +4.  When I print it, it shows +8.

I"ve attached the a pdf of the output and a screen shot of it in the program

![image](https://cloud.githubusercontent.com/assets/24965394/21739594/a3c776e0-d453-11e6-8f7b-e50296cf2abe.png)

[swansonstd.pdf](https://github.com/Sklore/HL_DD_5e_Colab/files/691265/swansonstd.pdf)

AD

**Comment by Sklore** [comment]
Ok i managed to replicate this. It's not the battle axe. It's coming from the FIghting Style which I'm guessing you picked Two-Weapon Fighting. 

That's not something we control though so I am raising this through to LW to look at.

**Comment by Sklore** [comment]
The feat is adding 4 to the damage bonus of the weapon, which Dual Wielder shouldn't be, according to the text. Assign it the Helper.TwoWpLigh tag to suppress the error from wielding a non-light weapon in the off-hand.
--
Hollis McCray

**Comment by TheEnthusiast** [comment]
@Sklore 

I introduced this one when we fixed the issue of the feat not adding the bonus to the offhand weapon, for non light weapons. 

I'm trying to fix it but so far I could not come up with a tag expression that hits all the right spots (non-light, non-two-handed, versatile in the off hand but not currently being two-handed and so on...)

I'm considering if it would be a bad thing to actually set the wProperty.Light on the weapons being wielded? It could cause issues with monk weapon selections or maybe the rogue's sneak attack. Or maybe I'm overthinking.

Thoughts?
